releases/v23_0_0.rst

```diff
@@ -13,16 +13,17 @@ Release 23.0.0 (2021-12-21)
 +-------------------------------------------+------------+
 | Source                                    | Identifier |
 +===========================================+============+
-| Git tag                                   | 23.0.0     |
+| Git tag                                   | 23.0.1     |
 +-------------------------------------------+------------+
-| :doc:`EUPS distrib </install/newinstall>` | v23\_0\_0  |
+| :doc:`EUPS distrib </install/newinstall>` | v23\_0\_1  |
 +-------------------------------------------+------------+
 
 
-This release is based on the ``w_2021_40`` weekly build, with 29 tickets backported to support the Data Preview 0.2 data release.
+This release is based on the ``w_2021_40`` weekly build (September 30, 2021), with 41 tickets backported to 23.0.1 to support the Data Preview 0.2 Data Release.
```

Comment:
If it's the case that 23.0.0 was used to do part of the release and that 23.0.1 was used to do another part, but none of the first part results would have been changed by 23.0.1, it might be nice to say that.